packages/config/src/layer2s/templates/opStack.ts

```diff
@@ -158,6 +157,19 @@ export function opStack(templateVars: OpStackConfig): Layer2 {
           : templateVars.finality,
     },
     chainConfig: templateVars.chainConfig,
+    dataAvailability:
+      templateVars.daProvider !== undefined
+        ? makeDataAvailabilityConfig({
+            type: 'Off chain',
+            layers: ['Celestia', 'Ethereum (calldata)'],
```

Comment:
In OP stack projects Celestia will always be the provider? Is it safe to use it here as default value?

Reply:
No